### DISCUSSION TITLE
Multi-Platform App (Uno Platform|net6) Win 10

### DISCUSSION THREAD

**Discussion by Software-Greve** [unoplatform/uno#8616] [Q&A]
Hello,
 
i have problems with the Visual-Studio project 

Multi-Platform App (Uno Platform|net6) 
Microsoft Visual Studio Community 2022 (64-bit) - Current Version 17.1.5
Win 10

Severity        Code    Description     Project File    Line    Suppression State
Error           Project "C:\(Entwicklung)\(SG)\Experimente\Video-Game-03\Video-Game-03\Video_Game_03.Shared\Video_Game_03.Shared.projitems" was not imported by "C:\(Entwicklung)\(SG)\Experimente\Video-Game-03\Video-Game-03\Video-Game-03.Mobile\Video-Game-03.Mobile.csproj" at (39,3), due to the file not existing.       Video-Game-03.Mobile    C:\(Entwicklung)\(SG)\Experimente\Video-Game-03\Video-Game-03\Video_Game_03.Shared\Video_Game_03.Shared.projitems       39
 
Severity        Code    Description     Project File    Line    Suppression State
Error   NU1012  Platform version is not present for one or more target frameworks, even though they have specified a platform: net6.0-android, net6.0-ios, net6.0-maccatalyst, net6.0-macos     Video-Game-03.Mobile    C:\(Entwicklung)\(SG)\Experimente\Video-Game-03\Video-Game-03\Video-Game-03.Mobile\Video-Game-03.Mobile.csproj  1
 
Do you know to resolve that issue?
 
Kind regards,
Ulrich

**Comment by jeromelaban** [comment]
Thanks for the report. Make sure to follow the directives here: https://github.com/unoplatform/uno/discussions/7833
